vuu-ui/packages/vuu-shell/src/layout-management/LayoutList.tsx

```diff
@@ -0,0 +1,51 @@
+
+import { List } from '@finos/vuu-ui-controls';
+import './LayoutList.css'
+
+export type LayoutMetadata = {
+    name: string,
+    group: string,
+    screenshot: string,
+    user: string,
+    date: string
+}
+
+export const LayoutsList = (props: { layouts: LayoutMetadata[] }) => {
+
+    const { layouts } = props;
+
+    const layoutsByGroup = layouts.reduce((acc: { [groupName: string]: LayoutMetadata[] }, cur) => acc[cur.group] ? { ...acc, [cur.group]: [...acc[cur.group], cur] } : { ...acc, [cur.group]: [cur] }, {})
```

Comment:
I think this is a bit long for a single-line ternary statement. Could we put it into a code block, and use some more descriptive variable names? Might make it a bit more transparent.